File: RolePlayGame/dotnet/src/my.examples/UserGateway.cs
using System.Collections.Generic;

namespace my.examples {

	public class UserGateway {
	
		public List<User> Teens() {
			List<User> teens = new List<User>();
			
			foreach (User eachUser in Db.AllUsers()) {
				if(eachUser.IsUnder(16)){
					teens.Add(eachUser);
				}
			}
			
			return teens;
		}
	}
}

File: RolePlayGame/java/src/my/examples/UserGateway.java
package my.examples;

import java.util.ArrayList;
import java.util.List;

public class UserGateway {

	public List<User> teens() {
		List<User> teens = new ArrayList<User>();
		
		for (User eachUser : Db.allUsers()) {
			if(eachUser.isUnder(16)){
				teens.add(eachUser);
			}
		}
		
		return teens;
	}
}
